developer-docs/bolt-server.md

```diff
@@ -223,12 +280,30 @@ bolt-server: {
 
 7. Start bolt-server
 `service pe-bolt-server start`
-8. Build a request
-Save the following JSON to `~/request.json`
+
+8. setup your environment for running commands with
```

Comment:
s/setup/Setup/ for consistency with other numbered items.

Reply:
it is for ValidationError